src/dom/DomEvent.js

```diff
@@ -263,23 +263,24 @@ export function getMousePosition(e, container) {
 	);
 }
 
+// @function getWheelPxFactor(): Number
+// Gets the wheel pixel factor based on the devicePixelRatio
+export function getWheelPxFactor() {
+	// We need double the scroll pixels (see #7403 and #4538) for all Browsers
+	// except OSX (Mac) -> 3x, Chrome running on Linux 1x
+	return (Browser.linux && Browser.chrome) ? window.devicePixelRatio :
+		Browser.mac ? window.devicePixelRatio * 3 :
+		window.devicePixelRatio > 0 ? 2 * window.devicePixelRatio : 1;

```

Comment:
Nit: I don't like the repetition.

```suggestion
	var ratio = window.devicePixelRatio;
	return Browser.linux && Browser.chrome ? ratio :
		Browser.mac ? ratio * 3 :
		ratio > 0 ? 2 * ratio : 1;
```